ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -147,69 +149,116 @@ const LocalStandardDisks = ({ idPrefix, onAddErrorNotification }) => {
         return <EmptyStatePanel loading />;
     }
 
+    const localDisksInfo = (
+        <Popover
+          bodyContent={
+              <div>{_(
+                  "Locally available storage devices (SATA, NVMe SSD, " +
+                  "SCSI hard drives, external disks, etc.)"
+              )}
+              </div>
+          }
+        >
+            <button
+              type="button"
+              onClick={e => e.preventDefault()}
+              className="pf-c-form__group-label-help"
+            >
+                <HelpIcon noVerticalAlign />
+            </button>
```

Comment:
It is taken from https://www.patternfly.org/v4/components/form#form-group-with-additional-label-info.

Reply:
I played a bit with that and it's achievable using the `Button` component with `variant="plain"`, but the style seems to be needed to align it correctly anyway, and the aria stuff is probably also there for a reason, so that saves nothing.

Reply:
Created a new component.